app/views/provider_interface/reports/diversity_reports/show.html.erb

```diff
@@ -0,0 +1,94 @@
+                                                                       'Applied',
+                                                                       'Offered',
+                                                                       'Recruited',
+                                                                       'Percentage recruited'],
+                                                             rows: @diversity_report_ethnicity_data,
+                                                             show_footer: true,
+                                                             exclude_from_footer: ['Percentage recruited'],
+                                                             bold_row_headers: false) %>
+
+      <h2 class="govuk-heading-m">Age</h3>
+      <%= render ProviderInterface::ReportTableComponent.new(headers: ['Age group',
+                                                                       'Applied',
+                                                                       'Offered',
+                                                                       'Recruited',
+                                                                       'Percentage recruited'],
+                                                             rows: @diversity_report_age_data,
+                                                             show_footer: true,
+                                                             exclude_from_footer: ['Percentage recruited'],
+                                                             bold_row_headers: false) %>
+
+      <h2 class="govuk-heading-m">About this data</h2>
+        <p class="govuk-body">
+          This data comes from candidates who submitted an application from <%= CycleTimetable.apply_opens.to_fs(:govuk_date) %>.
+        </p>
+        <p class="govuk-body">
+          The sex, disability and ethnicity data comes from <%= @completed_e_and_d_survey_count %> candidates who filled in a questionnaire when they applied. This is <%= (@completed_e_and_d_survey_count / @total_submitted_applications) * 100 %>% of the total candidates.
+        </p>
+        <p class="govuk-body">
+          The age data is from all <%= @total_submitted_applications %> candidates. It's based on each candidate’s age on <%= Time.zone.today.to_fs(:govuk_date) %>.
+        </p>
+
+      <h3 class="govuk-heading-s">How candidates are asked about their ethnicity</h3>
+      <p class="govuk-body">
+        Candidates are asked to select a general ethnic group, such as 'Asain or Asian British'. They can also select 'prefer not to say'.
+      </p>
+      <p class="govuk-body">
+        If the candidate selects a general ethnic group then they can select a more specific background such as 'Bangladeshi'. They can also select 'prefer not to say'.
+      </p>
+      <h3 class="govuk-heading-s">How candidates are asked about their disabilities</h3>
```

Comment:
Can we move this to before the ethnicity paragraph so it follows the same order as the tables?

Reply:
🆗